launch.rb

```diff
@@ -11,12 +11,8 @@ module FastlaneCI
   class Launch
     class << self
       include FastlaneCI::Logging
-
-      attr_accessor :build_queue
```

Comment:
neat, it looks like `add_build_runner` takes care of queuing up stuff, that's great!